packages/select/eslint.config.js

```diff
@@ -0,0 +1,9 @@
+const config = require("../../eslint.config.js");
+const tseslint = require("typescript-eslint");
+
+module.exports = tseslint.config([
+    config,
+    {
+        ignores: ["*.js", "*.d.ts"],
```

Comment:
Added this configuration to replace the `.eslintignore` file since ESLint no longer uses it: https://eslint.org/docs/latest/use/configure/migration-guide#ignoring-files